x/shared/session.go

```diff
@@ -5,6 +5,10 @@ import (
 	sharedtypes "github.com/pokt-network/poktroll/x/shared/types"
```

Comment:
@red-0ne Do we have a mermaid diagram anywhere that shows the windows & params?

I'd advocate to have it be done now because even I'm personally beginning to get confused and lose track of what's going on.